docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md

```diff
@@ -48,3 +44,11 @@ For Studio on Cloud, only the instance creator and the Nebula Graph Cloud Servic
 Authentication is not enabled in Nebula Graph by default. Users can log into Studio with the `root` account and any password.
 
 When Nebula Graph enables authentication, users can only sign into Studio with the specified account. For more information, see [Authentication](../../7.data-security/1.authentication/1.authentication.md).
+
+## Check updates
+
+Studio is in development. Users can view the latest releases features through [Changelog](../about-studio/st-ug-release-note.md).
+
+To view the Changelog, on the upper-right corner of the page, click the version and then **New version**.
+
+![On the upper right corner of the page, click Version and then New Version](https://docs-cdn.nebula-graph.com.cn/figures/st-ug-001-en.png)
```

Comment:
Minimize the screenshotting area to control the size of the pictures. Take only the part that users need to recognize.

Example:
![image](https://user-images.githubusercontent.com/69501902/160522801-aa20a739-2bf3-4315-a340-74942b80bbd1.png)